aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/translator/resourcemodel/ReadResourceModelTranslator.java

```diff
@@ -59,4 +63,30 @@ public static ResourceModel translateToResourceModel(final GetPatchBaselineRespo
         return model;
     }
 
+    /**
+     * Translate Read Response to Resource Model
+     */
+    public static ResourceModel translateToResourceModel(final DescribePatchBaselinesResponse describePatchBaselinesResponse,
```

Comment:
this is for List, i think you can create a separate file for it, or you change the name of this file

Reply:
Now we have reverted the changes in ReadResourceModelTranslator and created the new file for listhandler resouce model operations